utils/task_executor.py

```diff
@@ -1,46 +1,45 @@
 import logging
 from utils import config, helpers
 from integrations import browser, computer, discord, facebook, google, lam_at_home, open_interpreter, telegram
+from utils import contact_parse
 
 def execute_task(context, text):
     words = text.split()
-    if len(words) < 2:
+    if len(words) < 3:
         logging.error("Command did not provide enough parameters.")
         return
     
-    integration = words[0].strip('.,!?:;"').lower()
-    recipient = words[1].strip('.,!?:;"').lower()
-    message = ' '.join(words[2:]).strip()
+    integration, recipient, action, message = contact_parse.process_message(words)
```

Comment:
Not sure this is the best design for parsing the contacts, will discuss more with you in VC